ROADMAP.md

```diff
@@ -1,99 +1,51 @@
 # ROADMAP
 
-The kops 1.N.x version officially supports kubernetes 1.N.x and earlier.  While kubernetes 1.99 will likely run with kops 1.98,
-the configuration will probably not be correct (for example docker versions, CNI versions etc).
+## VERSION SUPPORT
+kops 1.N.x _officially_ supports Kubernetes 1.N.x and prior versions. We understand that those in the community run a wide selection of versions and we do our best to maintain backward compatibility as far as we can. 
 
-kops 1.N.0 is released when it is believed that kubernetes 1.N.x is stable, along with all the core addons (e.g. networking).
-This can mean that kops can release months after the release of kubernetes.  It's also not a deterministic release criteria,
-particularly with some networking plugins that are supported by kops but themselves still under development.  We discussed
-this challenge in kops office hours in March 2018, and the consensus was that we want to keep this, but that we should release
-alphas & betas much earlier so that users can try out new kubernetes versions on release day.
+However, kops 1.N.x does NOT support Kubernetes 1.N+1.x. Sometimes you get lucky and kops 1.N will technically install a later version of Kubernetes, but we cannot guarantee or support this situation. As always, we recommend waiting for the official release of kops with minor version >= the version of Kubernetes you wish to install. Please see the [compatibility matrix](README.md#Compatibility_Matrix) for further questions.
```

Comment:
I still suggest wrapping lines at a reasonable line width (there seemed to be better line wrapping before this change)

Reply:
@willthames I appreciate the suggestions. My git-fu is apparently failing me. I've ended up all turned around due to test flakes yesterday and now I'm in a weird place :) rebasing and squashing and we'll see if we can get this back on track!

As for the line wraps- I've generally let the browser/github deal with that. Is this paragraph displaying in a strange way for you? It looks reasonable to me- but if you can help me improve it, please do. I'm open to suggestions, I just don't know what you're looking for exactly. Maybe a screenshot would help!
Thanks again

Reply:
@geojaz - shorter lines tend to (but don't always) make for smaller diffs and simpler reviews (the fewer words on a line, the easier it is to say which line has the typo).

It's also slightly easier to get to a word in a text editor when using the keyboard as you can get to the right line first and then the right word.

It's displaying fine in github - I've definitely come across times where that's not the case (I seem to recall bitbucket doesn't do so well, but that's not an issue for this repo)

Reply:
@willthames Cool, I'm picking up what your putting down... understood. i'll come back to this tomorrow with fresh eyes and see what I can do for ya.